code/game/objects/items/RCD.dm

```diff
@@ -9,6 +9,8 @@ ARCD
 */
 
 /obj/item/construction
+	name = "not for ingame use"
```

Comment:
I got yelled at for doing stuff like this once

Reply:
This is one where it is certainly not and has overrides for all children already.